### DISCUSSION TITLE
resx example

### DISCUSSION THREAD

**Discussion by artmedia** [oqtane/oqtane.translations#2] [General]
Hello,
is there an example of .resx to start with?

**Comment by hishamco** [comment]
Hi @artmedia, I will post the resx files for Arabic in upcoming few days, but if there's a direct question feel free to ask

**Reply by artmedia** [reply]
@hishamco I will wait for your files, so I can understand how to start building one for the Italian language, tnx

**Reply by hishamco** [reply]
Ok, I already did some progress few weeks ago, but I'm busy with other PRs in two OSS projects, hope to push the resx in tiny PRs

**Reply by PavelVsl** [reply]
@hishamco are you able produce a default set of resx files in English? It will help a lot with oqtane translations.

**Reply by hishamco** [reply]
I started yesterday a tool to achieve that, seems I will publish the Arabic resx first or once I finish the tool the default language (English) will be here that contains the default localization, so everyone can copy paste the resx and change the values

**Comment by hishamco** [comment]
FYI I added the default resources (English), I will start to localize another language to make sure that nothing breaks, you could start to localize your favorite language too

**Comment by hishamco** [comment]
@artmedia you can start now with your language translations

**Reply by artmedia** [reply]
ok, tnx

**Reply by sbwalker** [reply]
@artmedia is it clear to you how to create a translation for Oqtane based on the current instructions and resource files? If not, please provide feedback on what you do not understand so that we can clarify and make it easier for people to get started.
